main/EDInitMod.F90

```diff
@@ -280,6 +281,7 @@ subroutine set_site_properties( nsites, sites )
           sites(s)%dstatus = dstat
           
           sites(s)%acc_NI     = acc_NI
+          sites(s)%NF         = 0.0_r8         
```

Comment:
@rgknox just to confirm: you are saying to remove this line?

Reply:
I think this is fine, I don't think it will have an effect. 
Looks like you already fixed the line I was talking about.

Reply:
OK. I was a little unclear on what was necessary in EDInit, so happy to take your input on if it needs to be changed.